dev/grunt_options/glotpress_download.js

```diff
@@ -0,0 +1,15 @@
+/**
+ *
+ * Module: glotpress
+ */
+
+module.exports = {
+  core: {
+    options: {
+      url: 'http://translations.tri.be',
+      domainPath: 'lang',
+      slug: 'the-events-calendar',
+      textdomain: 'tribe-events-calendar',
+    },
+  },
```

Comment:
Yeah, that's right. Even though trailing commas aren't in the JSON spec, doesn't mean we can't use them in config files. Keep 'em!